src/docsBrowser.ts

```diff
@@ -33,16 +34,40 @@ export namespace DocsBrowser {
         panel = window.createWebviewPanel('haskell.showDocumentationPanel', ttl, ViewColumn.Beside, {
           localResourceRoots: [Uri.parse(documentationDirectory)],
           enableFindWidget: true,
+          // TODO: a separate PR (window's content not destroyed when goes out of sight)
+          // retainContextWhenHidden: true,
+          enableCommandUris: true,
         });
         const uri = panel.webview.asWebviewUri(Uri.parse(path));
-        panel.webview.html = `<iframe src="${uri}" frameBorder = "0" style = "background: white; width: 100%; height: 100%; position:absolute; left: 0; right: 0; bottom: 0; top: 0px;" /> `;
+        const encodedPath = encodeURIComponent(JSON.stringify(path));
+        // TODO : better markup / divs?
+        panel.webview.html = `<table>
+          <tr><td><a href="command:haskell.openDocumentationInBrowser?${encodedPath}">Open In Browser</a></td></tr>
+          <tr><td><iframe src="${uri}" frameBorder = "0" style = "background: white; width: 100%; height: 100%; position:absolute; left: 0; right: 0; bottom: 0; top: 30px;"/></td></tr>
+        </table>`;
       } catch (e) {
         window.showErrorMessage(e);
       }
       return panel;
     });
   }
 
+  export function registerDocsOpenInBrowser(): Disposable {
+    return commands.registerCommand('haskell.openDocumentationInBrowser', async (path: string) => {
+      try {
+        // when opened in default program, file URL ignores the # anchor tag at shows from the top
+        // (at least on Windows) so must specify the browser explicitly
+        // TODO : move chrome to the settings
+
+        // open file in browser and close the webview in VS code
+        await openPath(path, { app: 'chrome' });
```

Comment:
Can we use the built-in vscode open functionality instead so we can avoid the extra dependency on `open`?
```typescript
import { env } from 'vscode';
// ...
env.openExternal(vscode.Uri.parse(path));
```

Reply:
didn't know about it, will try, thanks

Reply:
OK , env.openExternal suffers from the same issue as open called with default program, it strips anchor/tag in file url which is annoying

e.g. URL passed to openExternal:

file:///C:/sr/snapshots/b84f27c3/doc/haskell-lsp-types-0.22.0.0/src/Language.Haskell.LSP.Types.Location.html#Range

URL opened in browser (opens at top):

file:///C:/sr/snapshots/b84f27c3/doc/haskell-lsp-types-0.22.0.0/src/Language.Haskell.LSP.Types.Location.html

will check later if it can be done without pulling the 'open' dependency